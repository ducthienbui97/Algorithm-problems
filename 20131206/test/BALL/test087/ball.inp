9 677170 140154 785340 878910
595785 809673 545493 66790 844164 496122 438096 355197 984029 354478 606753 130251 483616 697368 55856 896101 56934 684669 785507 629263 654497 463967 261273 134613 726929 358373 962501 677794 941900 488236 922106 467377 73208 619108 315772 625548 903739 670789 444356 955583 837332 710466 787919 476958 25259 230395 289077 360964 202338 810064 684105 167369 441474 452790 87072 659649 40551 225049 254583 59736 873151 337928 83719 785447 29106 413513 819544 781817 892716 630998 957425 179912 589425 580564 506572 393646 572185 655852 292212 26075 99000 
894975 897299 315701 552804 115950 716499 790372 361140 933272 68604 680624 501547 3628 553372 831157 725134 358808 854674 837827 764466 465698 97263 446430 158628 779153 114172 474355 335175 932104 278341 292450 653208 354657 152901 327992 903515 516430 37484 968058 327523 861874 145707 704019 571056 25228 535822 215288 882087 523105 10098 300114 206750 51212 166221 17364 166099 673704 879949 886610 178254 337924 751714 901304 974006 436727 319671 362889 640444 631956 943301 382490 327570 250175 220655 743009 160525 400527 49542 364446 345318 931769 
153739 353972 914937 49251 801085 6987 554012 550699 252043 52052 142506 465281 40399 947871 199981 991913 845550 900828 740327 778154 770489 40855 261351 13577 445426 99406 447242 667648 609248 728458 754619 860712 6781 46316 492209 665616 585815 146471 513760 348007 503793 845006 917957 450277 622401 252283 183906 500713 301394 165865 533412 23218 351143 827079 209357 983366 699464 307955 198090 320757 746052 896138 759134 717695 405346 907927 310846 892834 71376 491097 65223 249470 284740 573721 981161 292791 872762 521846 821727 602797 390752 
2964 438160 745156 160094 857710 942281 10017 843161 603738 909577 154267 187718 663002 215745 922640 961606 558333 208465 186053 14084 187817 587113 624485 427246 879140 304134 272372 128269 716422 361320 942348 20270 126962 441073 127363 343442 354948 129580 296721 29275 584480 820686 254349 523111 152799 670741 283581 562410 543747 803754 539487 522369 595690 991943 297090 657689 867337 207073 176549 566072 658923 546483 470116 457574 197043 694884 500685 127325 898935 216688 442666 436444 250854 928411 211999 109543 596903 624188 297594 409884 614342 913769 71916 487746 788972 375951 931798 198272 356693 408288 481740 918589 86899 821185 508340 43987 817262 414199 645190 932774 261798 659651 737985 814038 410296 126997 230628 830660 945357 370150 812887 458784 13613 479018 555848 9473 35096 463794 464804 422687 983586 206820 127713 787204 521564 594290 944987 634849 575095 34548 7470 34211 828404 65208 116977 340148 694146 554586 320800 917441 855185 899218 219665 11095 498111 630666 943102 996231 498760 319990 816831 981666 470276 968864 256827 485778 864429 954864 546559 742733 476027 67384 334179 53428 573248 958795 83766 202315 699310 500094 399512 876607 473896 368352 279996 64076 905415 44685 782274 162691 257272 152629 899450 832700 335068 641619 64628 557875 370341 2452 227482 285372 28624 819996 217972 738454 456338 603546 661179 192903 203587 777935 688116 42974 5809 403491 502272 169862 502893 360902 384248 379047 732016 917213 212301 805955 707506 833217 388435 283894 579951 671848 644599 349903 857405 687799 220875 491972 487496 51856 687727 420973 317288 905530 505725 49086 686136 647290 858760 91 71241 530883 678397 425518 112641 541432 720609 211839 725234 414770 651274 595573 994392 512333 826091 134053 453426 817003 48619 379799 48356 418351 118865 427456 963274 225115 821044 230074 937382 386352 824735 632718 395899 555126 84679 20148 46955 863664 352701 403361 235648 584495 229330 964479 697544 557683 218712 74194 963195 55038 219225 450968 29224 592897 56558 62243 207153 31535 150444 3656 560936 321638 404725 774976 719407 873634 399362 128292 299392 96001 646294 377222 209341 615893 365630 63418 482932 486691 953571 309034 55319 476035 1052 586415 219799 920152 115959 532670 704489 2533 527821 804985 882917 642903 771834 735937 569077 90387 71051 853385 964049 501161 210756 500482 576025 223671 379306 628784 147786 335077 973207 451567 362746 739260 665381 170279 364655 805584 769885 472491 3351 925111 954299 957302 175533 487451 20315 489872 862705 791418 849138 238552 679820 432174 997115 15244 687705 454866 599059 276817 166939 908597 684722 148818 98505 113112 25723 575349 265278 344445 564397 860551 741203 782092 35321 978905 570105 72894 512949 598549 755129 525891 277839 598680 10935 307117 967127 338171 407146 106134 287779 171650 634156 243083 738411 482405 413339 673268 894194 701824 232291 393387 941164 727357 315912 319155 54800 356999 992371 810688 213561 425834 613543 652308 444184 880769 922783 646364 675543 159028 238412 215508 484703 537082 114339 347084 663572 336317 167965 400607 26031 734555 767692 643421 831110 612281 223663 64838 110420 218849 192233 997060 694732 827351 540106 16631 27450 950244 607264 993912 724004 868727 729230 414124 908819 627982 480762 505079 581390 959111 579061 814116 107334 135186 277925 692859 782263 896858 45459 976069 145012 516668 629483 246233 720501 499970 253743 93998 665563 635445 330108 214508 537771 203243 673197 491263 746985 666116 75731 570056 859280 504053 159568 954594 821341 240115 168182 257743 111316 55982 770710 819676 757349 740853 695789 192120 267328 683944 708508 27083 283806 942400 789677 605954 561548 522446 729771 589001 916862 122522 971254 273003 209812 112048 18784 941676 741481 103632 106857 242678 851280 411574 602069 489559 44444 301257 898558 423005 839790 841263 741535 450952 804513 466310 414567 602914 446190 297280 827577 60686 222615 460455 750891 379546 320760 161926 138024 747556 899595 104411 551953 399380 325313 873918 949353 590460 731804 591167 549801 546780 14180 148521 965687 88658 923246 294510 490409 811091 981855 349726 574175 6038 972395 95456 407160 56550 176360 676657 233308 630618 242975 678177 506030 565933 840123 979344 369379 125274 891748 240856 808925 863383 114987 986733 578818 373373 818912 71876 926492 93538 479144 78201 393374 21768 538850 733975 615069 812165 172713 91776 305041 791835 280947 572305 720091 977222 310899 172143 683809 330620 260725 971458 645493 827297 823155 223063 218528 722329 627950 170127 541986 931114 103152 158818 444708 787514 279870 5675 268024 908644 336119 425515 244648 299859 825674 997590 415938 197695 273395 261280 116522 155000 469714 91092 544314 757463 798453 808435 472986 573383 386898 759551 653788 528286 569903 607595 934956 781481 565873 344795 35241 103878 477068 625057 221481 194737 828052 98092 10878 785160 933756 833872 64994 66649 322171 635060 938496 691764 353736 281882 791144 984129 717897 399291 897691 185009 240689 61695 768807 