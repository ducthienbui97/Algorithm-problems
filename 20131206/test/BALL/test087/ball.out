16852985070550
